platformio.ini

```diff
@@ -52,3 +52,5 @@ board = esp07s
 
 build_dir = build/esp07s
 
+[env:d1_mini]
+board = d1_mini
```

Comment:
shouldn't you add a build_dir for that section as well? 
it uses it then to store firmwares for each board